users/utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+from core.settings import EMAIL_USER
+
+
+class Util:
+    @staticmethod
+    def send_email(data: Any) -> None:
+        email_body = render_to_string(
+            "reset_password.html", {"body": data.get("body")}
```

Comment:
make it dynamic by not setting the template, you can just accept the body in data and send it

Reply:
Added the template name as a function parameter instead so it is dynamic depending on the template passed